lib/fbe/middleware/formatter.rb

```diff
@@ -105,4 +120,16 @@ def dump_headers(headers)
     return '' if headers.nil?
     headers.map { |k, v| "#{k}: #{v.inspect}" }.join("\n")
   end
+
+  # Truncates text if it is longer than a specified :length by appending :omission
+  #
+  # @param [String, nil] txt The text to truncate
+  # @param [Integer] length The number of characters to keep
+  # @param [String] omission The string that will be added to the end as an omission
+  # @return [String] The truncated text, or an empty string if input was nil
+  def truncate(txt, length: 100, omission: '...')
```

Comment:
@Yegorov you can use this gem instead: https://github.com/yegor256/ellipsized

Reply:
@yegor256 exactly, thanks!